config/puma.rb

```diff
@@ -1,3 +1,26 @@
 # frozen_string_literal: true
 
 plugin :statsd
+
+# Configure "min" to be the minimum number of threads to use to answer
+# requests and "max" the maximum.
+#
+# The default is "0, 16".
+#
+threads 32, 32
+
+# === Cluster mode ===
+
+# How many worker processes to run.
+#
+# The default is "0".
+#
+workers 2
+
+# Verifies that all workers have checked in to the master process within
+# the given timeout. If not the worker process will be restarted. Default
+# value is 60 seconds.
+#
+worker_timeout 60
+
```

Comment:
Usually you'd see `rackup DefaultRackup` here to ensure puma uses `config.ru`, but now that we're on puma 4.x, its probably not necessary.